web/src/package.json

```diff
@@ -137,6 +136,7 @@
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "@cypress/webpack-preprocessor": "4.0.2",
+    "@material-ui/pickers": "^3.1.0",
```

Comment:
Let's keep direct dependencies explicit (drop the `^`) based on the issues we found yesterday.